fluentbit-template.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: logcollector
+objects:
+#
+# Deployment of fluentbit log collector
+#
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: fluentbit
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: fluentbit
+    template:
+      metadata:
+        labels:
+          app: fluentbit
+      spec:
+        containers:
+        - image: ${fluentbit_image}
+          securityContext:
+            privileged: true
+            allowPrivilegeEscalation: true
+          imagePullPolicy: Always
+          name: fluentd
+          command: ["/usr/local/bin/sh"]
+          args:
+          - -c
+          - >
+            rm /var/log/containers/stress.log;
+            /fluent-bit/bin/fluent-bit -c /etc/fluentbit/fluentbit.conf -R /etc/fluentbit/fluentbit.parsers.conf;
+            while true; do
+              echo ".";
+              sleep 60;
+            done;
+          volumeMounts:
+          - name: varlog
+            mountPath: /var/log
+          - name: config-path
+            mountPath: /etc/fluentbit
+        volumes:
+        - name: varlog
+          hostPath:
+            path: /var/log
+        - name: config-path
+          configMap:
+            name: fluentbit-config
+        restartPolicy: Always
+parameters:
+  - name: fluentbit_image
```

Comment:
There is default value from bash level, here: https://github.com/ViaQ/cluster-logging-collector-benchmarks/pull/1/files#diff-44133797f573b7ceda048bb2dc56353ef30a40de72ffdfb7afc6cd5754d77339R11

The logic for is to print the configuration before deployment ... I want all configuration information to be sent out to stdout so users will know what exactly runs in the test 

BTW: I had to use the `fluent-bit:1.7-debug` image because I wanted to delete leftovers (i.e. /var/log/containers/stress.log ) from previous runs as part of the deployment process